### ISSUE TITLE
Short term memory impairment

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6768]
**Preferred term label:**
Short term memory impairment

**Synonyms**
Short term memory loss

**Definition (free text, please give PubMed ID)**
A deficit in the retention of pieces of information (memory chunks) for a relatively short time (usually up to 30 seconds).

Comment:
The impairment of short term memory (STM) involves forgetting information to which the subject has been recently exposed. An individual with signs of losing STM, indeed, asks for the same questions repeatedly, forgets where he just put something, forgets recent events or something he saw or read recently. The loss of immediate memory is also termed as fixation amnesia. Practically, STM works as a kind of “scratchpad” for temporary recall of a limited number of data that come from the sensory register and are ready to be processed through attention and recognition. On the other side, information collected in the long term memory (LTM) storage consist of memories for the performance of actions or skills (i.e., procedural memories, knowing how) and memories of facts, rules, concepts, and events (i.e., declarative memories, knowing that). 
PMID:31424720
PMID:18394484

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0002354

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
